srcs/layouts/deva_phonetic_in.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<keyboard name="हिन्दी फोनेटिक - Hindi Phonetic" script="devanagari">
+  <modmap>
+    <shift a="अ" b="आ" />
```

Comment:
I added these to `shift.json` in bd2170e. You should be able to merge the master branch and remove most of them except for these three: ड़ क़ ज्ञ (that can't work in shift.json for now)

Reply:
I have removed them. I think everything looks good. Most of the those shifted to `shift.json` would be useful for other future layout  if any but some may be odd. Existing Hindi layouts don't use `<modmap>` so no need to worry I think.